Author: Steven Chamberlain <steven@pyro.eu.org>
Description: The missing sys/stat.h prototype leads to an implicit declaration
 of fchmod(), which uses signed int while kFreeBSD expects unsigned int, which
 results in a wrong bitmap, which leads to a build failure.
 This is only required for the fig2mpdf patch.
Bug-Debian: http://bugs.debian.org/654767

--- a/fig2dev/dev/genpstex.c
+++ b/fig2dev/dev/genpstex.c
@@ -47,6 +47,7 @@
 #include <stdlib.h>
 #include "bool.h"
 
+#include <sys/stat.h>
 #include "fig2dev.h"
 #include "object.h"	/* does #include <X11/xpm.h> */
 #include "texfonts.h"
